### PR TITLE
Update oauth-protection.md

### PR DESCRIPTION
**Pull request by manojs1978**
### Prepare

- [ ] Read [PR guidelines](https://github.com/JanssenProject/jans/blob/main/docs/CONTRIBUTING.md#prs)
- [ ] Read [license information](https://github.com/JanssenProject/jans/blob/main/LICENSE)

-------------------

#### Target issue
  
  
  
  
closes #issue-number-here

#### Implementation Details
  

-------------------
### Test and Document the changes
- [ ] Static code analysis has been run locally and issues have been fixed
- [ ] Relevant unit and integration tests have been added/updated
- [ ] Relevant documentation has been updated if any (i.e. user guides, installation and configuration guides, technical design docs etc)

### CODE REVIEW

**Review by mo-auto — APPROVED** [review]

**Review by ossdhaval — APPROVED** [review]

File: docs/admin/scim/oauth-protection.md [hunk 1]
```diff
@@ -2,16 +2,24 @@
...
+  
+  -Locate the Scim properties section
+  
+  -Select OAUTH in the Protection Mode list
+  
+  -Click the "Save Configuration" button at the bottom
+  
 ## Have questions in the meantime?
 
 While this documentation is in progress, you can ask questions through [GitHub Discussions](https://github.com/JanssenProject/jans/discussion) or the [community chat on Gitter](https://gitter.im/JanssenProject/Lobby). Any questions you have will help determine what information our documentation should cover.
```
Comment (ossdhaval) [inline docs/admin/scim/oauth-protection.md]: You can remove this section too as you have added content now.

File: docs/admin/scim/oauth-protection.md [hunk 2]
```diff
@@ -2,16 +2,24 @@
 tags:
   - administration
   - scim
+  - oauth-protection
 ---
```
Comment (ossdhaval) [inline docs/admin/scim/oauth-protection.md:7]: Consider writing a couple of lines that explain what will be achieved by the steps below.

**Review by ossdhaval — CHANGES_REQUESTED** [review]

File: docs/admin/scim/oauth-protection.md (hunk 1 above)
Comment (manojs1978) [reply docs/admin/scim/oauth-protection.md]: removed

**Review by ossdhaval — APPROVED** [review]

**Comment by manojs1978** [comment]
alrady updated, these changes not needed

**Pull request closed without merging**